src/cmd.ts

```diff
@@ -13,27 +14,43 @@ var program = require('commander');
 // set up sdsAccess
 
 
-async function uploadAndRunAll(loginData: config.LoginData, folder: string, prefix: string): Promise<sdsAccess.scriptT[]> {
+async function uploadAndRunAll(loginData: config.LoginData, files: sdsAccess.scriptT[]): Promise<sdsAccess.scriptT[]> {
```

Comment:
Okay. Since you're the only one using it and it is not exported the change is fine. 👍